airmozilla/popcorn/static/popcorn/js/editor.js

```diff
@@ -1,2 +1,27 @@
+function getSlug() {
+  var pathname = window.location.pathname,
+      re = /\/popcorn\/edit\/(.+)\//,
+      match = pathname.match(re);
+
+  return match[1];
```

Comment:
This whole function ought to be refactored so it doesn't depend on the URL. 
Where I said to use a `data` attribute to transfer the value of the URL from Django to the javascript via a django template, do the same with the value of the slug.